### ISSUE TITLE
Support ASCII character for histograms

### ISSUE THREAD

**Issue by vlsi** [jlink/jqwik#376]
## Testing Problem

Here's an exctract from my Jenkins log (it is Windows machine)

```
0:14:37.629     timestamp = 2022-07-28T11:01:00.753550500, [...Properties:...] (1137) tables = 
00:14:37.629            # | label | count | 
00:14:37.629         -----|-------|-------|---------------------------------------------------------------------------------
00:14:37.629            0 |     0 |   145 | ???????????????????????????
00:14:37.629            1 |     1 |   335 | ??????????????????????????????????????????????????????????????
00:14:37.629            2 |     2 |   429 | ????????????????????????????????????????????????????????????????????????????????
00:14:37.629            3 |     3 |   158 | ?????????????????????????????
00:14:37.629            4 |     4 |    60 | ???????????
00:14:37.629            5 |     5 |    10 | ?
```

It would be nice to use a character that is representable on a default Windows console

**Comment by jlink** [comment]
Any suggestion as for which character is universal across platforms and still looks somewhat like a square?
Maybe it should be configurable?

**Comment by vlsi** [comment]
It looks like the only sane way is so have something trivial (e.g. `X`) until Windows terminal gets better: https://stackoverflow.com/a/17177904/1261287
For instance: `if (System.getProperty("os.name").toLowerCase().contains("win")) { 'X' }`

**Comment by jlink** [comment]
I'll use "#" as fallback char.

**Comment by jlink** [comment]
Fixed in https://github.com/jlink/jqwik/commit/985e5ddf8ef8b3a8fa5e4fd6183298a61d358094.

Released in 1.7.1-SNAPSHOT
